unit-tests/unit-tests-live.cpp

```diff
@@ -3823,6 +3823,108 @@ TEST_CASE("Per-frame metadata sanity check", "[live][!mayfail]") {
     }
 }
 
+TEST_CASE("color sensor API", "[live][options]")
```

Comment:
It is better to rearrange the test so that there will be a concentrated map of all the sensors extensions per PID for clearness and maintenance instead of having separate tests per sensor type.